### PR TITLE
Red lock token fix

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/25313

## Description
In multi-environment instances, the customer name is a must in order to log in. Added docs pointing that out and error handling in case there is no customer.

## Screenshots
Settings 2020-06-04 21-28-03

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/RedLock/Integrations/RedLock/RedLock.py [hunk 1]
```diff
@@ -33,9 +33,18 @@ def get_token():
         'username': demisto.getParam('credentials')['identifier'],
         'password': demisto.getParam('credentials')['password']
     })
+
     if response.status_code != requests.codes.ok:
         raise Exception('Error authenticating to RedLock service [%d] - %s' % (response.status_code, response.text))
-    TOKEN = response.json()['token']
+    response_json = response.json()
```
Comment (Itay4) [inline Packs/RedLock/Integrations/RedLock/RedLock.py:39]: lets wrap this in `try-except` to catch `ValueError` that might be raised

File: Packs/RedLock/Integrations/RedLock/RedLock.py [hunk 2]
```diff
@@ -33,9 +33,18 @@ def get_token():
...
+    TOKEN = response_json.get('token')
+    if not TOKEN:
+        message = 'Could not retrieve token from server: {}'.format(response_json.get("message"))
+        if response_json.get('message') == 'login_needs_customer_name':
+            available_customer_names = [name.get('customerName') for name in response_json.get('customerNames')]
+            message = 'In order to login a customer name need to be configured. Available customer names: {}'.format(
```
Comment (Itay4) [inline Packs/RedLock/Integrations/RedLock/RedLock.py:45]: lets add a unit test for this

**Comment by kirbles19** [comment]
@ShahafBenYakir Done.

**Comment by ShahafBenYakir** [comment]
Pairing this message - https://panw-global.slack.com/archives/G011E63JXPB/p1591446287439200 force merging this PR.

**Pull request merged**